docs/source/apex.rst

```diff
@@ -41,7 +46,7 @@ Enable 16-bit
 .. code-block:: python
 
     # turn on 16-bit
-    trainer = Trainer(amp_level='O1', precision=16)
+    >>> trainer = Trainer(amp_level='O1', precision=16)  # doctest: +SKIP
```

Comment:
if is it skipped there is no need to test it, but why skip this?

Reply:
yes I agree. If I turn it back to sphinx doctest we can explicitly skip only if the machine does not have amp. I have the sphinx stuff on a different branch, I will force push it here then I can finishe it for review, ok?